pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -185,8 +196,8 @@ func (w *Webhook) inject(pod *corev1.Pod, ns string, _ dynamic.Interface) (bool,
 		}
 	}
 
-	libsToInject, autoDetected := w.extractLibInfo(pod)
-	if len(libsToInject) == 0 {
+	extractedLibInfo := w.extractLibInfo(pod)
+	if len(extractedLibInfo.libs) == 0 {
 		return false, nil
```

Comment:
If this line is hit, the following 2 injections (security client & profiling client) will be ignored. 

Please make sure if this is the intended behaviour (i.e. security client config & profiling client config should not be injected if APM injected no tracing libraries)

Reply:
Will check-- this PR isn't changing this behavior though-- prior behavior would _also_ have aborted injection in this case.

If it's not intended, we can fix in a follow up.